python/doc/theory/reliability_sensitivity/reliability_index.rst

```diff
@@ -1,44 +1,39 @@
 Reliability Index
 -----------------
 
-| The generalized reliability index :math:`\beta` is used under the
-  following context: :math:`\vect{X}` is a probabilistic input vector,
-  :math:`\pdf` its joint density probability, :math:`\vect{d}` a
-  deterministic vector, :math:`g(\vect{X}\,,\,\vect{d})` the limit state
-  function of the model,
-  :math:`\cD_f = \{\vect{X} \in \Rset^n \, / \, g(\vect{X}\,,\,\vect{d}) \le 0\}`
-  the event considered here and g(,) = 0 its boundary.
-| The probability content of the event :math:`\cD_f` is :math:`P_f`:
-
-  .. math::
-    :label: PfX6
+The generalized reliability index :math:`\beta_g` is used under the
+following context: let :math:`\vect{X}` be a probabilistic input
+vector with joint density probability  :math:`\pdf`, let
+:math:`\vect{d}` be a  deterministic vector, let :math:`g(\vect{X}\,,\,\vect{d})` be the limit state function of
+the model and let :math:`\cD_f = \{\vect{X} \in \Rset^n \,
+/ \, g(\vect{X}\,,\,\vect{d}) \le 0\}` be an event which probability
+:math:`P_f` is defined as:
 
-      P_f = \int_{g(\vect{X}\,,\,\vect{d}) \le 0}  \pdf\, d\vect{x}.
+.. math::
+    :label: PfX6
+    
+    P_f = \int_{g(\vect{X}\,,\,\vect{d}) \le 0}  \pdf\, d\vect{x}.
 
 The generalized reliability index is defined as:
 
 .. math::
 
     \beta_g = \Phi^{-1}(1-P_f) = -\Phi^{-1}(P_f).
 
-| As :math:`\beta_g` increases, :math:`P_f` decreases rapidly.
 
-These indices are available:
+As :math:`\beta_g` increases, :math:`P_f` decreases rapidly.
 
--  :math:`\beta_{FORM}` the FORM reliability index, where :math:`P_f` is
-   obtained with a FORM approximation (refer to ~): in this case, the
-   generalized reliability index is equal to the Hasofer-Lindt
-   reliability index :math:`\beta_{HL}`, which is the distance of the
-   design point from the origin of the standard space,
+According to the method used to evaluate :math:`P_f`, the generalized reliability index differs:
 
--  :math:`\beta_{SORM}` the SORM reliability index, where :math:`P_f` is
-   obtained with a SORM approximation : Breitung, Hohenbichler or Tvedt
-   (refer to ),
+-  when :math:`P_f` has been obtained from the :doc:`form_approximation` approximation, then 
+:math:`\beta_g` is equal to the  Hasofer-Lindt reliability index :math:`\beta`, which is the 
```

Comment:
This should fix the build error: `reliability_index.rst:29:Bullet list ends without a blank line; unexpected unindent`:
```suggestion
  :math:`\beta_g` is equal to the  Hasofer-Lindt reliability index :math:`\beta`, which is the 
```

Reply:
ok. I modified the text.